work/flow.rst

```diff
@@ -511,7 +520,6 @@ This history makes ``git bisect`` searches more obvious.
 We understand that this workflow may take longer to get used to, but the end result of a clean history has paid us dividends in finding when particular algorithmic changes had unexpected effects.
 
 .. figure:: /_static/development/git-rebase-history.png
-   :target: ../_images/git-rebase-history.png
```

Comment:
What's wrong with keeping this?  It allows people to expand the image if desired.

Reply:
This was an attempt to fix the broken linkcheck step in the Weekly CI. The more recent runs seem to be broken on other things, making me think that this is flaky. I'll exclude this commit from the PR.

https://github.com/lsst-dm/dm_dev_guide/actions/runs/5239410991/jobs/9459211675